src/azul/indexer/aggregate.py

```diff
@@ -163,6 +163,31 @@ def get(self):
         return thaw(super().get())
 
 
+class SetOfListOfDictsAccumulator(SetOfDictAccumulator):
+    """
+    A set accumulator that supports lists of mutable mappings as values.
+    """
+
+    def __init__(self, max_size=None, key=None, filter_=None):
+        """
+        :param filter_: A callable that can be used to filter the values that
+                        are accumulated.
+        """
+        super().__init__(max_size, key)
+        self.filter_ = filter_
+
+    def accumulate(self, value) -> bool:
+        assert isinstance(value, list)
+        if self.filter_:
+            value = filter(self.filter_, value)
+        all_statuses = True
+        for v in value:
+            assert isinstance(v, dict)
+            if not super().accumulate(v):
+                all_statuses = False
+        return all_statuses
```

Comment:
This is a completely optional alternative which I wrote just for fun.
```suggestion
        assert all(isinstance(v, dict) for v in value)
        return all(super().accumulate(v) for v in value)
```

Reply:
@jessebrennan yeah i considered this, i was torn w/ looping twice vs the longer block of code......

Reply:
attempted this, does not work as expected...

Reply:
I had to change it to 
```python
        assert all(isinstance(v, dict) for v in value)
        return all(super(SetOfListOfDictsAccumulator, self).accumulate(v)
                   for v in value)
```
I'm confused about WHY the type , its as if the interpreter is confused about how to resolve super.

The error when I was using what you suggested was 

```python
  File "/Users/amar/dev/hca/azul/src/azul/indexer/aggregate.py", line 420, in _accumulate
    accumulator.accumulate(value)
  File "/Users/amar/dev/hca/azul/src/azul/indexer/aggregate.py", line 184, in accumulate
    return all(super().accumulate(v)
  File "/Users/amar/dev/hca/azul/src/azul/indexer/aggregate.py", line 184, in <genexpr>
    return all(super().accumulate(v)
TypeError: super(type, obj): obj must be an instance or subtype of type
```